corehq/apps/linked_domain/applications.py

```diff
@@ -67,15 +67,15 @@ def get_downstream_app_id(downstream_domain, upstream_app_id, use_upstream_app_i
 
 
 def get_upstream_app_ids(downstream_domain):
-    return list(_get_downstream_app_id_map(downstream_domain))
+    return list(get_downstream_app_id_map(downstream_domain))
 
 
 @quickcache(
     vary_on=['downstream_domain', 'use_upstream_app_id'],
     skip_arg=lambda *args, **kwargs: settings.UNIT_TESTING,
     timeout=5 * 60
 )
-def _get_downstream_app_id_map(downstream_domain, use_upstream_app_id=False):
+def get_downstream_app_id_map(downstream_domain, use_upstream_app_id=False):
```

Comment:
I could have sworn I listened to your advice and made arguments like `use_upstream_app_id` default to True since that is what we want going forward. I must be thinking of a different method though. Not requesting any change as that is out of scope of this PR.

Reply:
7c790dbe295d8bf083a9feea5bf33607a7e4f752

Reply:
Easy enough. Thank you!